migrate-to-chimera

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+arg=$1
+if [ $# -gt 1 ] ; then
+    echo "This script get at most one argument"
+    echo "Usage:"
+    echo "$0 [-l|--logout]"
+    echo ""
+    echo "  -l | --logout   Logout after migration"
+fi
+
+if ! [ -d $HOME/.config/steam-buddy ] && \
+    ! [ -d $HOME/.local/share/steam-buddy ] && \
+    ! [ -d $HOME/.cache/steam-buddy ] ;
+then
+    # Nothing to migrate, we can exit
+    echo "Found nothing to migrate"
+    exit 0
+fi
+
+## MIGRATION MAGIC ##
+
+# Check if there are config files from steam-buddy
+if [ -d $HOME/.config/steam-buddy ] ; then
```

Comment:
Migrating `.config/steam-buddy`, `.local/share/steam-buddy`, `.cache/steam-buddy` should be done at the very end.
If the script errors out after migrating these then re-running will never trigger the later migrations.

i.e. those folders are an indicator that we are done the migration

Reply:
The order should be:
- settings.json
- config files
- content under share
- shortcuts
- change all references in all files

But each migration already checks if the source directory is there. Why it would be a problem?

Reply:
Imagine this case:
 - the script starts, migrates .config/steam-buddy, .local/share/steam-buddy, .cache/steam-buddy successfully
 - the script fails or is killed in the next step so nothing else is migrated
 - now start the script again, it checks the directories, sees .config/steam-buddy, .local/share/steam-buddy, .cache/steam-buddy are all migrated and then makes the decision to exit and nothing else is ever migrated

Reply:
So you mean, we create the new chimera directory structure inside steam-buddy one and then simply move it?

Or just change the references inside the files first and then migrate all the directories? Suggested order:
- settings.json
- change all references in all files
- config files
- shortcuts
- content under share

Deleting each leftover after directory migration.

Reply:
I think that would work? Alternatively, do not exit the script early and make the decision to migrate each individual piece independently.

Reply:
Ok. then we test each individual piece and dont quit at the beggining? That may need to add a flag to test if we needed to migrate anything. I think the best way is to test if there are leftovers and if there are not, just migrate. maybe we could add a `force` switch to skio that check?

Reply:
Changed the order so `.local/share` is migrated last after settings by mv